### ISSUE TITLE
Issue024: stability analysis doesn't work for O atom in 3-21G.

### ISSUE THREAD

**Issue by ndattani** [HPQC-LABS/QCBugs#23]

**Comment by ndattani** [comment]
@deyanmihaylov have you searched online about the warnings and errors that you got? After doing a generic search in a search engine, have you thoroughly checked MMSE and the PySCF GitHub page for all closed and open issues and all closed and open pull requests?

**Comment by deyanmihaylov** [comment]
@ndattani yes, I have searched on Google, I have searched on MMSE and I have searched the repository without finding anything. I had searched on Google even before reporting the problem in the Discord thread.

**Comment by ndattani** [comment]
> @ndattani yes, I have searched on Google, I have searched on MMSE and I have searched the repository without finding anything. I had searched on Google even before reporting the problem in the Discord thread.

Have you ***thoroughly*** searched ***all*** of those places? Have you searched "all closed and open issues and all closed and open pull requests" ?

**Comment by deyanmihaylov** [comment]
I did search both the issues and the pull requests. I found one issue which mentions the same error, but doesn't provide any details:
https://github.com/pyscf/pyscf/issues/1726

I could not find any `pyscf` PRs relating to this.

**Comment by ndattani** [comment]
@deyanmihaylov  have you also searched all "closed" issues and pull requests? It is a bit frustrating asking the same question three times, but when your response literally leaves out key details, this is the only way that I can be certain of the answer to the first question.

**Comment by ndattani** [comment]
@deyanmihaylov you searched both "the issues and pull requests", but for each of those did you search both the "open ones and the closed ones"?
